tardis/transport/montecarlo/packet_trackers.py

```diff
@@ -265,14 +266,12 @@ def finalize_array(self):
         """
         Added to make RPacketLastInteractionTracker compatible with RPacketTracker
         """
-        pass
```

Comment:
This may look odd, but apparently a docstring counts as an empty statement and thus `pass` is not required.